recipes/chart/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "Chart.js" %}
+{% set version = "2.7.1" %}
+{% set sha256 = "e9e39d3124221cc7666f6ef2cb0d00e07f682929cc90abba36c1b76f27014f7c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/chartjs/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: generic
+  script: npm install -g .
+
+requirements:
+  build:
+    - nodejs
+  
+test:
+  commands:
+    - npm list -g chart.js
+  requires:
+    - nodejs
+   
+
+about:
+  home: https://github.com/chartjs/Chart.js
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: 'Simple HTML5 Charts using the <canvas> tag'
+  description: |
+    Simple HTML5 Charts using the <canvas> tag  
+  doc_url: http://www.chartjs.org/
+  dev_url: https://github.com/chartjs/Chart.js
+
+extra:
+  recipe-maintainers:
+    - cshaley
```

Comment:
@cshaley -- can you please confirm that you are ok with being added to this and the other js recipes?

Reply:
Yes and any other package @sannykr adds me to

Reply:
I don't think that it is. Node is just needed to get npm to install the package